src/library_fs.js

```diff
@@ -414,17 +414,23 @@ FS.staticInit();` +
           },
           isAppend: {
             get: function() { return (this.flags & {{{ cDefine('O_APPEND') }}}); }
-          }
+          },
+          get flags() { return this.shared.flags; },
+          set flags(value) { this.shared.flags = value; },
+          get position() { return this.shared.position; },
+          set position(value) { this.shared.position = value; },
```

Comment:
The other get/set methods on this prototype use a different syntax (that avoid duplicating the property name).  Any reason not to do that same here?  At least we should probably be consistent.

Reply:
Did you see this comment?

Reply:
Should be fixed.